onnx/backend/test/case/node/celu.py

```diff
@@ -14,10 +14,12 @@ class Celu(Base):
 
     @staticmethod
     def export():  # type: () -> None
+        alpha = 2
```

Comment:
Shouldn't it be alpha=2. for the tests to pass?

Reply:
Hmm weird -- I must have somehow missed including the actual source when I fixed it in 65fc77f7a84a3dc15d26bfd70175c2ced7289984 :man_facepalming: Good catch!